contrib/build-wheel.sh

```
@@ -84,7 +90,11 @@ if [ "$CUDA_MAJOR" -ne 12 ] && [ "$CUDA_MAJOR" -ne 13 ]; then
 fi
 PKG_NAME="nixl-cu${CUDA_MAJOR}"
 ./contrib/tomlutil.py --wheel-name $PKG_NAME pyproject.toml
-uv build --wheel --out-dir $TMP_DIR --python $PYTHON_VERSION
+if [ "$BUILD_NIXL_EP" = "true" ]; then
+    uv build --wheel --out-dir $TMP_DIR --python $PYTHON_VERSION -Csetup-args=-Dbuild_nixl_ep=true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)